docs/reference/buildx_history_logs.md

```diff
@@ -14,3 +14,34 @@ Print the logs of a build
 
 <!---MARKER_GEN_END-->
 
+## Description
+
```

Comment:
"without multiplexing." probably doesn't need to be mentioned. That is more like an implementor's note.